ExampleLegacyAlerts.swift

```diff
@@ -0,0 +1,27 @@
+//
```

Comment:
This file has been added in the project root.  It needs to be moved to the correct source directory.